source/_integrations/roomba.markdown

```diff
@@ -33,7 +33,7 @@ This platform has been tested and is confirmed to be working with the iRobot Roo
 
 <div class='note'>
 
-The Roomba's MQTT server only allows a single connection. Enabling continuous mode will force the App to connect via the cloud to your Roomba. [More info here](https://github.com/NickWaterton/Roomba980-Python#firmware-2xx-notes)
+The Roomba's MQTT server only allows a single connection. If another smart home platform uses continues mode, Home Assistant will not be able to connect to your Roomba. Enabling continuous mode in Home Assistant will force the App to connect via the cloud to your Roomba. [More info here](https://github.com/NickWaterton/Roomba980-Python#firmware-2xx-notes)
```

Comment:
I'm confused, this was already implied by the first sentence, right?

> The Roomba's MQTT server only allows a single connection.

So what makes "another smart home platform" different?

Reply:
Yet, it is implied, but it took me a very long time to connect the dots. The error I got caused me to go down the wrong path. Now that I know what the problem was, it is obvious from the documentation, but it was not before. I expect there are more users who have the same experience. 

My use case:
I'm migrating from one smart home platform to another. For nearly all devices, Home assistant can monitor the device while the ohter platform is controlling it. I can then switch over the automation for a seamless experience. 

With Roomba, this strategy didn't work. I thought the other platform connected to Roomba via the cloud. When I read the documentation, I didn't realize that other platforms would also use continues mode by default. 

The error I got from Home Assistant was a password error. When using the Roomba980 script, I got a connection error half-way through the process. Googling for these error did not point me towards the continues connection issue, so i though it would be useful to mention it more explicitly and/or to relate the (somewhat misleading) error message to the specific problem.  

This is my first edit, so let me know if there is a better place or format I should use.

Reply:
Thanks for willing to contribute, but this text is already in place. It is just added now twice again in a different working, causing the same warning now to be on this page 3 times.

I think that is overkill and thus going to friendly decline this change.